client/app/pages/queries/view.js

```diff
@@ -120,6 +120,7 @@ function QueryViewCtrl(
     Notifications.getPermissions();
   };
 
+  $scope.dataSourceHasNotChanged = true;
```

Comment:
Let's turn around the logic and call this `dataSourceChanged` with a default of `false` so we don't have to deal with a double negative below.

Reply:
@jezdez Updated.

Reply:
@alison985 Thank you! 😃